trace-normalization/src/normalizer.rs

```diff
@@ -103,11 +113,71 @@ pub(crate) fn is_valid_status_code(sc: &str) -> bool {
     false
 }
 
+// normalize_trace takes a trace and
+// * returns an error if there is a trace ID discrepancy between 2 spans
+// * returns an error if at least one span cannot be normalized
```

Comment:
If you want comments to turn up in autocompletions, or rust documentation, each line should start with three [slashes](https://doc.rust-lang.org/rust-by-example/meta/doc.html#:~:text=Doc%20comments%20are%20very%20useful,a%20%2F%2F%2F%20%2C%20and%20support%20Markdown.&text=%2F%2F%2F%20%2F%2F%20If%20you%20pass,even%20test%20it%20for%20you!).

```suggestion
/// normalize_trace takes a trace and
/// * returns an error if there is a trace ID discrepancy between 2 spans
/// * returns an error if at least one span cannot be normalized
```